hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java

```diff
@@ -389,6 +395,19 @@ private void writeFilesToArchive(Set<Path> copyFiles,
     }
   }
 
+  @NotNull
+  private static Path getVerifiedCheckPointPath(Path checkpointLocation) {
+    Path metaDirPath = checkpointLocation;
+    // This check is done to take care of findbug else below getParent()
+    // should not be null.
+    Path locationParent = checkpointLocation.getParent();
+    if (null != locationParent) {
+      Path parent = locationParent.getParent();
+      metaDirPath = parent != null ? parent : locationParent;
```

Comment:
This should throw a null pointer exception.  switching it back to locationParent will cause a more confusing error.

Reply:
@GeorgeJahad - Thanks for your review. I have taken care of the comment. Pls re-review.